src/views/Projects/index.jsx

```diff
@@ -2,26 +2,49 @@ import { hot } from 'react-hot-loader';
 import { Component, Fragment } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import Grid from '@material-ui/core/Grid';
-import TextField from '@material-ui/core/TextField';
 import Typography from '@material-ui/core/Typography';
 import projects from '../../data/loader';
 import ProjectCard from '../../components/ProjectCard';
+import SearchBox from '../../components/SearchBox';
 
 @hot(module)
 @withStyles(theme => ({
   container: {
-    paddingRight: 15,
-    paddingLeft: 15,
+    padding: 15,
     marginRight: 'auto',
     marginLeft: 'auto',
+    backgroundColor: theme.palette.primary.light,
+    height: '100vh',
   },
   header: {
     paddingTop: theme.spacing.unit,
-    paddingBottom: 2 * theme.spacing.unit,
+    paddingBottom: 5 * theme.spacing.unit,
```

Comment:
Delete `paddingBottom` and replace with `height: '14vh',`.

Reply:
@helfi92 how about using a fixed height instead of percentage for the AppBar? i get this sometimes
<img width="1792" alt="screen shot 2018-06-20 at 1 14 46 am" src="https://user-images.githubusercontent.com/24544912/41613181-7e4f2b4a-7427-11e8-901d-28b35f00dfdc.png">

Reply:
Weird. Did you try adding the following style to the `container` class in addition to that:

```js
height: '86vh',
marginTop: `calc(14vh + ${3 * theme.spacing.unit}px)`,
```

Is it possible you forgot to add the `px` at the end?